docs/joss_paper/paper.md

```diff
@@ -67,12 +67,12 @@ Unsupervised learning was first developed independently from a number of researc
 ![Registration network with unsupervised loss.\label{fig:unsupervised}](../asset/deepreg-tutorial-unsupervised.svg)
```

Comment:
https://github.com/DeepRegNet/DeepReg/blob/274-joss-paper-finalising/docs/asset/deepreg-tutorial-unsupervised.svg

Link broken: since you use "../asset/deepreg-tutorial-unsupervised.svg" it will not resolve unless it points to master branch. Recommend change to master branch here.

Reply:
will check later - not sure the paper compilation takes https.